qemu.mk

```diff
@@ -32,6 +32,19 @@ all-clean: bios-qemu-clean busybox-clean linux-clean optee-os-clean \
 ################################################################################
 # QEMU
 ################################################################################
+QEMU_CONFIGURE_PARAMS = --target-list=arm-softmmu --cc="$(CCACHE)gcc" \
+						--extra-cflags="-Wno-error"
+
+QEMU_VIRTFS_ENABLE		?= n
```

Comment:
Add alias to `/etc/profile` in [gen_rootfs](https://github.com/linaro-swg/gen_rootfs/blob/master/etc/profile) and update the  [filelist-common](https://github.com/OP-TEE/build/blob/master/common.mk#L271) target with yet another directory (might not work with `tmp` btw, since sometimes `/tmp` is mounted using tmpfs, but another folder would do `/mnt/shared`).

```
alias mount_shared='mount -t 9p -o trans=virtio host /tmp/shared/'
```
Maybe this would be overkill, maybe just easier to add it to the README.md as Igor suggested in another comment. See this as suggestions and "not a must do thing" ;-)